src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/resource/DeleteAlertCodeIntTest.kt

```diff
@@ -0,0 +1,165 @@
+package uk.gov.justice.digital.hmpps.hmppsalertsapi.resource
+
+import org.assertj.core.api.Assertions
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.within
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.test.web.reactive.server.WebTestClient
+import uk.gov.justice.digital.hmpps.hmppsalertsapi.integration.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.hmppsalertsapi.integration.wiremock.TEST_USER
+import uk.gov.justice.digital.hmpps.hmppsalertsapi.integration.wiremock.USER_NOT_FOUND
+import uk.gov.justice.digital.hmpps.hmppsalertsapi.model.AlertCode
+import uk.gov.justice.digital.hmpps.hmppsalertsapi.model.request.CreateAlertCodeRequest
+import uk.gov.justice.digital.hmpps.hmppsalertsapi.repository.AlertCodeRepository
+import uk.gov.justice.digital.hmpps.hmppsalertsapi.utils.ALERT_CODE_VICTIM
+import uk.gov.justice.digital.hmpps.hmppsalertsapi.utils.ALERT_TYPE_CODE_VULNERABILITY
+import uk.gov.justice.hmpps.kotlin.common.ErrorResponse
+import java.time.LocalDateTime
+import java.time.temporal.ChronoUnit
+import java.util.UUID
+
+class DeleteAlertCodeIntTest : IntegrationTestBase() {
```

Comment:
Maybe better called DeactivateAlertCodeIntTest? I know the endpoint itself is a delete so either works. Minor anyway

Reply:
Done.